api/test/test_contact_group.py

```diff
@@ -44,3 +45,73 @@ def test_add_person_to_contact_group():
     assert r.status_code == 200
     # Assert that in API response success==true
     assert r.json['success']
+'''
+
+#ex01
+def test_add_organization_to_contact_group():
+    organization = add_organization("Test_org" + get_time_in_milliseconds())
+    organization_id = organization.json['data']['id']
+
+    contact_group = add_contact_group(group_name="org_group" + get_time_in_milliseconds(), group_type="organization",
+                                      description="Test group description")
+    contact_group_id = contact_group.json['data']['id']
+
+    endpoint = 'contactGroups/' + str(contact_group_id) + '/organizations'
+    payload = {
+        "item_ids": [organization_id]
+    }
+    r = do_request(method='POST', endpoint=endpoint, data=payload)
+
+    # Assert response code
+    assert r.status_code == 200
+    # Assert that in API response success==true
+    assert r.json['success']
+
```

Comment:
In addition to response check, it would be reasonable to do an additional request and verify the contents of the contact group.